src/crimson/os/seastore/root_block.h

```diff
@@ -0,0 +1,83 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include "crimson/os/seastore/cached_extent.h"
+
+namespace crimson::os::seastore {
+
+using depth_t = uint32_t;
```

Comment:
will uint_fast16_t be enough?

Reply:
Probably, but it's purely in-memory, so 32 bit is simpler.